extension/jmxmetricextension/README.md

```diff
@@ -66,9 +82,21 @@ The password to use for JMX authentication.
 
 Corresponds to the `otel.jmx.password` property.
 
-### otlp_timeout (default: 1s)
+### exporter (default: `otlp`)
+
+The metric exporter to use, which matches a desired existing `otlp` or `prometheus` receiver type.
+
+Corresponds to the `otel.exporter` property.
+
+### otlp_endpoint (default: `localhost:55680`)
 
-The otlp exporter request timeout.  Will be converted to milliseconds.
+The otlp exporter endpoint to submit metrics.  Must coincide with an existing `otlp` receiver `endpoint`.
+
+Corresponds to the `otel.otlp.endpoint` property.
+
+### otlp_timeout (default: `5s`)
```

Comment:
Are those the only two OTLP options? Can't you import the OTLP Settings here and support all options? I'm doing something similar here:

https://github.com/jpkrohling/opentelemetry-collector-contrib/blob/c661a0efc5c7c46e31cb1a73bfec067fb1bdee26/exporter/loadbalancingexporter/config.go#L23-L27

If it's not possible at all, then at least the `insecure` option should be added.

Reply:
These are for the otel-java sdk properties and don't map to OTLP receiver configs.  As is, a separate OTLP or prometheus receiver needs to be configured, though improved usability is being planned.